out/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -1,7 +1,7 @@
 [{if $edit && $edit->isCustomPaymentMethod()}]
   [{assign var="paymentMethod" value=$edit->getPaymentMethod()}]
   [{assign var="configFields" value=$paymentMethod->getConfigFields()}]
-[{/if}]
+  [{/if}]
```

Comment:
Why the indentation changes in this file?

Reply:
maybe because of the missing closing tags for the logo and description `<tr>`'s